root/io/stdpair/cmspair.h

```diff
@@ -0,0 +1,79 @@
+//
+// Odd pair:
+//   pair<reco::Muon::MuonTrackType,edm::Ref<vector<reco::Track>,reco::Track,edm::refhelper::FindUsingAdvance<vector<reco::Track>,reco::Track> > >
+// from map:
+//   map<reco::Muon::MuonTrackType,edm::Ref<vector<reco::Track>,reco::Track,edm::refhelper::FindUsingAdvance<vector<reco::Track>,reco::Track> > >"
+
+#if 0
```

Comment:
I assume this block is supposed to document something but it's not clear to me what it is.

Reply:
Yes, it is the distillation of the original problem (in full CMSSW) in a simple script.  I extended the comments.